fitbenchmarking/fitbenchmarking
# Organise structure of problems docs

#### Description of Work
Ensure that all problem sets have a descriptions with the following:
 - Download links
 - Information on source of data
 - Information on size of problem -- unknowns and data points.

Currently there is a folder on powell with new versions of everything in generated by:
```
for d in $(ls examples/benchmark_problems); do zip -r ${d}.zip $d; done
for d in $(ls examples/benchmark_problems); do tar -czvf ${d}.tar.gz $d; done
```

Fixes #1003 

#### Testing Instructions

1. Look at the files on powell (called `fitbenchmarking-new` in samelocation as live files)
2. Proof read docs
3. Check links appear to work -- note, this may need to be done after changing the folder name on powell

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
